website/src/controllers/user.js

```diff
@@ -117,12 +117,26 @@ api.score = function(req, res, next) {
     }, saved.toJSON().stats));
 
     // Webhooks
+
+    // Select character data to send
+    var userData = _.pick(user.toJSON(), ['_id', '_tmp', 'stats']); // user.toJSON to copy-by-value
```

Comment:
@paglias after testing further, I cannot simply `_.clone()` the property of the `user` object; `user` is a Mongoose model and not an actual JS object, and so when I try to clone it or mutate it it throws an error.

What I've done instead is factor out a similar line above that was calling the `.toJSON()` method to be reused by the webhooks.

`.toJSON()` is called a lot by the API endpoints. In order to send the object to the client, it pretty much has to be called.